darts/timeseries.py

```diff
@@ -1499,123 +1840,169 @@ def __rtruediv__(self, n):
         return n * (self ** (-1))
 
     def __abs__(self):
-        series = abs(self._df)
-        return TimeSeries(series, self.freq_str())
+        return TimeSeries(abs(self._xa))
 
     def __neg__(self):
-        series = -self._df
-        return TimeSeries(series, self.freq_str())
+        return TimeSeries(-self._xa)
 
-    def __contains__(self, ts: pd.Timestamp) -> bool:
-        return ts in self._df.index
+    def __contains__(self, ts: Union[int, pd.Timestamp]) -> bool:
+        return ts in self.time_index
 
     def __round__(self, n=None):
-        series = self._df.round(n)
-        return TimeSeries(series, self.freq_str())
+        return TimeSeries(self._xa.round(n))
 
-    def __lt__(self, other):
-        if isinstance(other, (int, float, np.integer, np.ndarray)):
-            series = self._df < other
+    def __lt__(self, other) -> xr.DataArray:
+        if isinstance(other, (int, float, np.integer, np.ndarray, xr.DataArray)):
+            series = self._xa < other
         elif isinstance(other, TimeSeries):
-            series = self._df < other.pd_dataframe()
+            series = self._xa < other.data_array(copy=False)
         else:
+            series = None
```

Comment:
Why is it set to None here?

Reply:
Again that's just for some IDEs not to complain... perhaps not ideal I agree. I'll remove it.